wiki/People/Beatmap_Nominators/en.md

```diff
@@ -32,24 +32,23 @@ Modders that aim to apply to the Beatmap Nominators must fulfil the following cr
 + The point in time at which the Kudosu has been awarded to you mod post does not matter. The only relevant information about a mod post is its post date and time in UTC+0 as well as the fact that it has been awarded Kudosu.
 
 ### How the Required Minimum Score is Calculated
-+ This depends on the total `Kudosu Score` and the total `Mod Score` an applicant achieved over the last months. A mathematical formula is used to determine these values.
-+ `Kudosu Score` is determined according to the following formula:
-
-![Kudosu Score Formula](/wiki/shared/KDSCORE.png "Kudosu Score Formula")
++ This depends on the total `Mod Score` an applicant achieved over the last months. This value is determined according to the following formula:
```

Comment:
Where did the formula go? Shouldn't the last sentence say `A mathematical formula is used to determine these values.`?